app/helpers/step_nav_actions_helper.rb

```diff
@@ -9,6 +9,13 @@ def can_submit_2i_review?(step_by_step_page, user)
       step_by_step_page.reviewer_id == user.uid
   end
 
+  def can_revert_to_draft?(step_by_step_page, user)
+    return true if step_by_step_page.status.approved_2i?
+
+    %w[submitted_for_2i in_review].include?(step_by_step_page.status) &&
+      step_by_step_page.review_requester_id == user.uid
```

Comment:
Is there are reason why only the user who submitted for 2i can revert to draft?

Reply:
As per Craig's comment:
1) the author should be able to Revert to draft
2) the reviewer should be able to Abandon review, changing it to submitted_for_2i. This is different functionality which is NOT part of this card. We have a separate card for this.
3) other people with review permissions should be able to Claim review, leaving status unchanged but changing the reviewer field.

Reply:
Could you add this to the commit message? and add a test for when the user is not the review requester?